coralproject/talk
# Configure stream settings

… query.

## What does this PR do?

Story Bug: https://www.pivotaltracker.com/story/show/143520971

- moves settings into the state
- adds premodlinks to the query

## How do I test this PR?

1) 
- Go to the 'Configure Stream' tab of the asset
- Start writing a question
- Wait
- Check that it does not disappear when the query happens

2) 
- Go to the 'Configure Stream' tab of the asset
- Enable or Disable Pre moderation links and Apply 
- Refresh
- Check that is still there what you changed.
